core/payment/examples/validate_allocation.rs

```diff
@@ -56,10 +58,12 @@ async fn main() -> anyhow::Result<()> {
     assert!(result.is_err());
     log::info!("Failed to create allocation (as expected).");
 
+    // We need to take ZkSync's transaction fees into account
+    let estimated_txs_cost = BigDecimal::from(20) * BigDecimal::from_str("0.033").unwrap();
```

Comment:
That's a time bomb. Today it works, tomorrow it might not. :(
Although I'm not sure how to modify the example to improve it.

Reply:
As we spoke - please check https://github.com/golemfactory/yagna/pull/1092/commits/fc3f4482e4afb878a93316b6dc10aceb0edc5bcb